src/alto-ui/SkipLink/__tests__/SkipLink.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import SkipLink from '../SkipLink'
+
+describe('SkipLink', () => {
+
+    beforeEach(() => jest.resetAllMocks());
+
+    const defaultProps = {
+        target: 'target',
+    }
+    
+    const getWrapper = props => shallow(<SkipLink {...defaultProps} {...props} />);
+    
+    it('should render without error',() => {
+        const wrapper = getWrapper({});
+        
+        expect(wrapper.find('.SkipLink').exists()).toBe(true);
+    })
+
+    it('should pass target props to a href attribute',() => {
+        const wrapper = getWrapper({});
+
+        expect(wrapper.find('.SkipLink').prop('href')).toBe('#target')
+    })
+
+    it('should call handleFocus function after button is clicked',() => {
```

Comment:
So if this function can be handle and tested than test all possibility of this function, not only one condition.
Check `element.tabIndex` for also falsy `originTabbindex` and some other number values.
Check also behavior for negative `element.tabIndex` value

You should also check if the element is focused!
For that I suggest don't mock `getElementById` but render element like below:
```
const getWrapper = (props, targetId = 'test-id') => shallow(<div><input id={targetId} /><SkipLink {...defaultProps} {...props} /></div>);
```

Reply:
Added all in `('Check tabIndex by click event', () => {` describe:

```
it('should have tabIndex === 0 as default', () => {
it('should not change tab index if bigger than 0', () => {
it('should be not focused', () => {
it('should be focused', () => {
```